policyengine_uk/utils/solve_private_school_attendance_factor.py

```diff
@@ -0,0 +1,41 @@
+from policyengine_uk import Microsimulation
+from policyengine_core.reforms import Reform
```

Comment:
we should not need to apply a structural reform if the program is part of the law